exporter.go

```diff
@@ -4,6 +4,10 @@
 package main
 
 import (
+	//We keep this at the top to ensure it happens first
```

Comment:
It'd be worth commenting _why_ the ordering is important, rather than stating what's being done here.